mypy/main.py

```diff
@@ -51,6 +51,13 @@ def main(script_path: Optional[str], args: Optional[List[str]] = None) -> None:
         args: Custom command-line arguments.  If not given, sys.argv[1:] will
         be used.
     """
+    # Check for known bad Python versions.
+    vi = tuple(sys.version_info)
```

Comment:
`sys.version_info` is already a tuple, so I think the `tuple` call is unnecessary.